hugo/themes/dora-team/assets/js/README.md

```diff
@@ -0,0 +1 @@
+Template js assets that are required for the DORA theme
```

Comment:
I think there's at least one js file which belongs here (`widgets.js`). Possibly others?

Reply:
Also (commenting here b/c I can't comment on a file that is renamed without changes 🫤), the `faq.scss` and `contact.scss` files feel like they belong to the specific site rather than the theme(?) -- WDYT?

Reply:
Yeah... there was a few of these "well, we might could support FAQs & Contacts in other sites.    Just let me know... easy moves